modules/aws/mlflow/main.tf

```diff
@@ -40,32 +40,26 @@ module "aws_iam_role_with_oidc" {
 }
 
 module "aws_iam_role_without_oidc" {
-  source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
-  version                       = "5.43.0"
-  create_role                   = true
-  role_name                     = "dvc-bucket-access-role"
-  custom_role_policy_arns       = [aws_iam_policy.this[1].arn, aws_iam_policy.this[2].arn]
-  trusted_role_arns             = ["arn:aws:iam::${local.account_id}:root"]
-}
-
-resource "kubernetes_namespace" "this" {
-  metadata {
-    name = "mlflow"
-  }
+  source                  = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
+  version                 = "5.43.0"
+  create_role             = true
+  role_name               = "dvc-bucket-access-role"
```

Comment:
are we naming it this way because this is the same role used for dvc bucket access?

Reply:
should this be mlflow artefact bucket access role or a generic name?